src/geowombat/ml/classifiers.py

```diff
@@ -188,6 +188,12 @@ def _mask_nodata(y, x, src_nodata=None, dst_nodata=np.nan):
             mask = np.any((x == src_nodata).values, 1, keepdims=True)
             return xr.where(mask, dst_nodata, y)
 
+        # suggested by jordan
```

Comment:
I've been hesitant to update the masking of values, but maybe I should

Reply:
I think I would leave it to the user to set `nodata` values and not do it by default. Unless you were to add an argument where `nodata` could be specified.